README.md

```diff
@@ -1,3 +1,273 @@
+    - email:                                # see above
+      username:                             # see above
+      hash:                                 # bcrypted hash of password, see above
+```
+
+Configures the identity provider that allows access to the Gardener dashboard. The easiest method is to provide a list of `users`, each containing `email`, `username`, and either a clear-text `password` or a bcrypted `hash` of the password.
+You can then login into the dashboard using one of the specified email/password combinations.
+
+## Most Important Commands and Directories
+
+### Commands
+
+These are the most important `sow` commands for deploying and deleting components:
+
+| Command&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Use              |
+|:------------------------------|:-----------------|
+| `sow help`| Displays a command overview for sow |
+| `sow order -a <component>` | Displays all dependencies of a given component (in the order they should be deployed in) |
+| `sow deploy -a <component>` | Deploys a component and all of its dependencies |
+| `sow delete -a <component>` | Deletes a component and all components that depend on it (including transitive dependencies) |
+|`sow deploy <component>`|Deploys a single component. The deployment will fail if the dependencies have not been deployed before. |
+|`sow <component>`| Same as `sow deploy <component>`.|
+|`sow delete <component>`|Deletes a single component|
+|`sow order -A`|Displays the order in which all components can be deployed|
+|`sow deploy -A`|Deploys all components in the order specified by `sow order -A`|
+|`sow delete -A`|Deletes all components in the inverse order|
+
+### Directories
+After using sow to deploy the components, you will notice that there are new directories inside your landscape folder:
+
+| Directory               | Use              |
+|:------------------------|:-----------------|
+| `gen`| Temporary files that are created during the deployment of components, e.g. generated manifests. |
+| `export` | Allows communication (exports and imports) between components.It also contains the kubeconfig for the virtual cluster that handles the Gardener resources. |
+| `state` | Important state information of the components is stored here, e.g. the terraform state and generated certificates. It is crucial that this directory is not deleted while the landscape is active. While the contents of the *export* and *gen* folders will be overwritten when a component is deployed again, the contents of *state* will be reused instead. In some cases, it is necessary to delete the state of a component before deploying it again, for example if you want to create new certificates for it.|
+
+#### Deleting Directories
+The `gen` directory can be safely deleted at any time, except *during* the deployment of a component.
```

Comment:
This is no longer true.

`gen` and `export` can be re-created from scratch by re-deploying the components, `gen` cannot. 

However, deleting a component might files that are created on deployment and stored in `gen`, so when the `gen` folder is deleted, it might be necessary to re-create it before it is possible to delete the component.

Also, all three folders (or their contents, at least) are deleted upon component deletion. Manually deleting them is only necessary in special cases.